src/components/assignments/AssignmentForm.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Form from 'grommet/components/Form';
+import FormField from 'grommet/components/FormField';
+import TextInput from 'grommet/components/TextInput';
+import DateTime from 'grommet/components/DateTime';
+import NumberInput from 'grommet/components/NumberInput';
+import CheckBox from 'grommet/components/CheckBox';
+import Button from 'grommet/components/Button';
+import Heading from 'grommet/components/Heading';
+import Footer from 'grommet/components/Footer';
+import Paragraph from 'grommet/components/Paragraph';
+
+const AssignmentForm = (props) => {
+  return (
+    <Form onSubmit={props.onSubmit} pad="medium">
+      <Heading tag="h2">{props.heading}</Heading>
+      <fieldset>
+        <legend><Paragraph size="small">Input the assignment details</Paragraph></legend>
+        <FormField htmlFor="name" label="Name" help="required">
+          <TextInput
+            id="name"
+            placeHolder="Assignment name"
+            onDOMChange={props.onChange}
+            required={true}
+            value={props.fields.name}
+          />
+        </FormField>
+        <FormField htmlFor="description" label="Description" help="required">
+          <textarea
+            id="description"
+            placeholder="Assignment description"
+            onChange={props.onChange}
+            required={true}
+            value={props.fields.description}
+          />
+        </FormField>
+        <FormField htmlFor="duedate" label="Due date" help="required">
+          <DateTime
+            id="duedate"
+            onChange={props.onChangeDate}
+            required={true}
+            value={props.fields.duedate}
+          />
+        </FormField>
+        <FormField htmlFor="classifications_target" label="Number of images per student" help="required">
+          <NumberInput
+            id="classifications_target"
+            min={1}
+            onChange={props.onChange}
+            required={true}
+            value={props.fields.classifications_target}
```

Comment:
I should make a note here before I forget to check:

The `classifications_target` value needs to be a string, correct? If so, would `<NumberInput>` submit the field's values to the `onChange()` function as a string or a number?

Reply:
Yeah, this one kind of boggles me. I figured out this should be a string after I begun this and it doesn't make sense, especially when it could be used to calculate stats like I'm trying to do in #90. Why is this a string instead of a number for the API?

Reply:
FWIW, I'll make it a string since that's what is being used with the API, I just find it strange.